docs/source/release_notes/release-notes.rst

```diff
@@ -91,14 +91,18 @@ develop
 
            So far, automated migration is only supported for Cassandra KVS.
            If using DBKVS or other key-value services, it remains the user's responsibility to ensure that they have performed the migration detailed in :ref:`Migration to External Timelock Services <timelock-migration>`.
-           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1569>`__, 
+           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1569>`__,
            `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1570>`__ and
            `Pull Request 3 <https://github.com/palantir/atlasdb/pull/1579>`__)
-           
+
     *    - |fixed|
          - Fixed multiple connection pool deadlocks in DbKvs
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1566>`__)
 
+    *    - |fixed|
```

Comment:
I just rebased, where's the update to `docs/source/release_notes/release-notes.rst` for 0.32.0?